katas/content/preparing_states/parity_bitstrings/solution.md

```diff
@@ -1,55 +1,55 @@
-There are multiple ways to approach this problem. In our first solution, we are going to use a recursive approach that was hinted at in the task.
+There are multiple ways to approach this problem. In your first solution, you're going to use a recursive approach that was hinted at in the task.
```

Comment:
Please rewrite to emphasize that they're reading about these solutions, not that we expect them to write all of them